modules/dnn/src/layers/slice_layer.cpp

```diff
@@ -78,32 +78,31 @@ class SliceLayerImpl CV_FINAL : public SliceLayer
         }
         else if (params.has("begin"))
         {
+            axis = params.get<int>("axis", 0);
```

Comment:
Can we get this argument once? There are already `axis = params.get<int>("axis", 1);` and `axis = params.get<int>("axis", 0);`. Does something works for any our tests?